airbyte-server/src/main/java/io/airbyte/server/handlers/WebBackendConnectionsHandler.java

```diff
@@ -80,14 +85,16 @@ public class WebBackendConnectionsHandler {
   private final SchedulerHandler schedulerHandler;
   private final OperationsHandler operationsHandler;
   private final EventRunner eventRunner;
+
   // todo (cgardens) - this handler should NOT have access to the db. only access via handler.
-  private final ConfigRepository configRepository;
+  @Deprecated
```

Comment:
The main issue I saw when trying that is that IntelliJ don't give feedback about this deprecated (No strike through or red color). That's why I went with an abstract class and getter and setter since using this field will look like an error. That being say, the new naming is pretty explicit so I'm fine with both solution.

Reply:
@benmoriceau yeah. i noticed it too. let's merge your PR since it gets the highlighting right. i'll merge mine after with the comment and the name change